packages/react-lowcode/src/codegen/interfaces.ts

```diff
@@ -39,13 +39,19 @@ export interface ColumnSourcePositionResult {
 
 export interface WidgetProperty {
     name: string,
-    value: string
+    value: string,
+    type: WidgetPropertyValue
 }
 
 export interface WidgetProperties {
     properties: WidgetProperty[]
 }
 
+export enum WidgetPropertyValue {
```

Comment:
Do we need text representations of items of this enum?

Reply:
I thought it will be better in demo/wysiwyg repository. Instead of having type `0|1` to have `EXPRESSION|STRING_LITERAL`. It's more clear i would say. But i can change that.